index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portifolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body> <!-- Header start-->
+    <header>
+        <div class="nav-container">
+            <a href="#" class="logo">WELCOME</a>
+            <nav>
+                <button class="menu" type="button" >
+                    <img src="images/menu.gif" alt="a menu link">
+                </button>
+            </nav>
+        </div>
+    </header>   <!-- Header end-->
+
+     <!-- main start-->
+    <section class="main">
+        <div class="intro">
+            <h1>Hey there. I’m Yordi ! </h1>
+            <h1><span class="a-span">I’m a software developer</span></h1>
+        </div>
+        <div class="header-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="media-link">
+            <ul class="socials">
+                <li><a href="#"><img src="images/twitter.png" alt="twitter image"></a></li>
+                <li><a href="#"><img src="images/Linkedin icon.png" alt="linkedin image"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="medium image"></a></li>
+                <li><a href="#"><img src="images/github.png" alt="github image"></a></li>
+                <li><a href="#"><img src="images/angellist.png" alt="angellist image"></a></li>
+            </ul>
+        </div>
+        <div class="down-btn">
+            <button class="btn" type="button">
+                <img class="img-btn" src="images/Action.png" alt="">
+            </button>
+        </div>
+    </section>
+    <section class="mycontr">
+        <h1 class="myrecentworks">My Recent Works</h1>
+        <img class="dash" src="images/Dash.png" alt="">
+        <div class="work-container">
+            <div class="img-container">
+                <img src="images/image.png" alt="A display">
+            </div>
+            <div class="work-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                    <li><a href="#">html</a></li>
+                </ul>
+                <button class="disp-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
```

Comment:
In my opinion, I think you don't need to put 'See project' inside the `span` tag in order to make your code dry.